client/signup/steps/theme-selection/index.jsx

```diff
@@ -95,27 +95,46 @@ class ThemeSelectionStep extends Component {
 		);
 	}
 
+	headerText() {
+		const { flowName, translate } = this.props;
+
+		if ( this.isStoreSignup() ) {
+			return translate( 'Choose a store theme.' );
+		} else if ( flowName === 'test-fse' ) {
+			return translate( 'Pick your site design' );
+		}
+		return translate( 'Choose a theme.' );
+	}
+
+	subHeaderText() {
+		const { flowName, translate } = this.props;
+
+		if ( this.isStoreSignup() ) {
+			return translate( 'Pick one of our store themes to start with. You can change this later.', {
+				context: 'Themes step subheader in Signup',
+			} );
+		} else if ( flowName === 'test-fse' ) {
+			return translate( "You'll be able to customize your new site in hundreds of ways.", {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?